### ISSUE TITLE
Ubuntu 20.04: error loading shared library: Dynlink.Error

### ISSUE THREAD

**Issue by dmitryhryppa** [EcsoKit/ecso#23]
Hello! 
I've just installed `ecso` via `haxelib`. Then after including it into the project I've got a compilation & completion error:

```
/home/dmitryhryppa/haxelib/ecso/1,0,0-alpha,4/hx/ecso/_core/Plugin.macro.hx:36: characters 5-10 : Uncaught exception [ECSO] Failed to load plugin: error loading shared library: Dynlink.Error (Dynlink.Cannot_open_dll "Failure(\"/home/dmitryhryppa/haxelib/ecso/1,0,0-alpha,4/cmxs/hx-4.2.2/Linux/plugin.cmxs: undefined symbol: camlHashtbl__6\")")
/home/dmitryhryppa/haxelib/ecso/1,0,0-alpha,4/hx/ecso/_core/Plugin.macro.hx:28: characters 3-14 : Called from here
(unknown) : Called from here
```

Did I miss something with my setup? 
Thank you! 

---
Haxe 4.2.2
ecso: [1.0.0-alpha.4]
Ubuntu 20.04

**UPD**: Just tried ecso on Windows 10 and it works fine there.

**Comment by dpomier** [comment]
I assume this can happen when building Haxe from source, in which case you should also [build ecso from source](https://github.com/EcsoKit/ecso/wiki/Installation#building-from-source). Is that your case?

**Comment by dmitryhryppa** [comment]
Not exactly. I've only updated my installed earlier official deb package (via `apt`, not from the source) 4.2.1 to 4.2.2. So after that `haxe --version` gives me `4.2.2`. 
But, to check it again I just tried official Haxe binaries, and it works! While installed deb package still provides an error. Interesting... Maybe the deb package was built with another version of OCaml or something like that?

Well, looks like this is not an esco issue, so I will close it.
Thank you! :)
